src/components/WebexMeetingGuestAuthentication/__snapshots__/WebexMeetingGuestAuthentication.stories.storyshot

```diff
@@ -89,7 +92,7 @@ Array [
             <input
               aria-errormessage=""
               aria-invalid="false"
-              aria-label="The password is provided in the invitation for a scheduled  meeting, or from the host."
+              aria-label="The password is provided in the invitation for a scheduled meeting, or from the host."
```

Comment:
Can we put it in a constants file?

Reply:
Since this is a snapshot file for testing, I dont think we can use a constant in snapshot and in the jsx file.